playground/AspireEventHub/EventHubs.AppHost/Program.cs

```diff
@@ -0,0 +1,18 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddAzureProvisioning();
```

Comment:
Can you generate and check in a manifest and bicep file for the playground? You can do this by:

```shell
cd playground/AspireEventHubs/EventHubs.AppHost
dotnet run -- --publisher manifest --output-path aspire-manifest.json
```